.github/workflows/update-banners.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Banners
+
+on: 
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  update-banners:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: actions/setup-node@v2.2.0
+        with:
+          node-version: '10.x'
+
+      - uses: actions/cache@v2.1.6
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: npm ci
+
+      - name: Check banner dates
+        run: npm run update-banners
+
+      - name: Check for changes
+        id: changes
+        uses: UnicornGlobal/has-changes-action@v1.0.11
+      
+      - name: Create PR
+        if: steps.changes.outputs.changed == 1
+        uses: peter-evans/create-pull-request@v3
```

Comment:
Ive never used this before, so im not positive what other options would be needed. Im still digging in [here](https://github.com/marketplace/actions/create-pull-request)

Reply:
Not relevant anymore.